Ramda/Extensions/dec.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  dec.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 28/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Decrements its argument.
+
+     - parameter value: Number to add to
+
+     - returns: Number + 1
+
+     See also inc
+
+     */
+    public class func dec<A: BasicArithmeticType>(_ number: A) -> A {
+
+        guard let one = 1 as? A else {
```

Comment:
I think we can add a method to `BasicArithmeticType` that has a `unit` property or just a `one` property that returns back the unit value for that type?

Reply:
Oh I noticed in the `inc` method you use the `.one` value on `A`, so you can just use that